module/tools/SystemConfiguration/src/SystemConfiguration.cpp

```diff
@@ -53,6 +53,38 @@ namespace module::tools {
         return output.str();
     }
 
+    void symlink(const fs::path& link, const fs::path& target, const NUClear::PowerPlant& powerplant){
```

Comment:
If you're only using powerplant for logging you can just do `NUClear::log()`
Or I suppose make this a member function